master-firmware/tests/error.cpp

```diff
@@ -0,0 +1,95 @@
+#include <CppUTest/TestHarness.h>
+#include <CppUTestExt/MockSupport.h>
+
+#include "error/error.h"
+
+TEST_GROUP(ErrorLoggingTestGroup)
```

Comment:
I would rename `ErrorLoggingTestGroup` to `ErrorLogging` as the "TestGroup" doesn't really adds information and decreases readability.